Makefile

```diff
@@ -16,10 +16,16 @@ VERSION=1.10.1
 # Setup the -ldflags option for go build here, interpolate the variable values
 #LDFLAGS=-ldflags "-X github.com/ariejan/roll/core.Version=${VERSION} -X github.com/ariejan/roll/core.BuildTime=${BUILD_TIME}"
 
-all:	init-config-host keymasterd-bin-data
-	cd $(GOPATH)/src; go install -ldflags "-X main.Version=${VERSION}" github.com/Cloud-Foundations/keymaster/cmd/*
+all:	init-config-host cmd/keymasterd/binData.go
+	cd cmd/keymaster; go install -ldflags "-X main.Version=${VERSION}"
+	cd cmd/keymasterd; go install -ldflags "-X main.Version=${VERSION}"
+	cd cmd/keymaster-unlocker; go install -ldflags "-X main.Version=${VERSION}"
+	cd cmd/keymaster-eventmond;  go install -ldflags "-X main.Version=${VERSION}"
```

Comment:
Is it not possible to maintain the wildcard approach?

Reply:
no if we want to ensure that there is no depenencies on having a "correct" directory mapping. This is not very elegant but ensures that the installed binaries actually came from the current source repository.